src/components/ModalDialog/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './style.css';
+
+class ModalDialog extends React.Component {
+    constructor(props){
+        super(props);
+        this.container = document.createElement('div');
+        document.body.appendChild(this.container);
+    }
+
+    componentWillUnmount(){
+        document.body.removeChild(this.container);
+    }
+
+    render(){
+        return ReactDOM.createPortal(
```

Comment:
ReactDOM.createPortal , can reference to https://segmentfault.com/a/1190000011668286